v1/types/types.gunk

```diff
@@ -251,3 +251,50 @@ type Location struct {
 	// The longitude in degrees. It must be in the range [-180.0, +180.0]
 	Longitude float64 `pb:"2" json:"longitude"`
 }
+
+// Structure of account information
+type ProfileAccountInfo struct {
+
+	// Bank code
+	BankCode BankCode `pb:"1" json:"bank_code"`
+
+	// Account identifier
+	AccountID string `pb:"2" json:"account_id"`
+
+	// Account identifier
+	ProfileID string `pb:"3" json:"profile_id"`
+}
+
+// Structure of customer profile information
+type Profile struct {
+
+	// ProfileID is the unique identifier of a profile.
+	ProfileID string `pb:"1" json:"profile_id"`
```

Comment:
can we just make id ID?
it would be profile.ID

Reply:
I thought of make it as ID, but referred to account proto, it was AccountID so I followed the same.